src/frontend/src/flows/recovery/inputSeedPhrase.ts

```diff
@@ -15,21 +19,20 @@ const pageContent = () => html`
       font-weight: 400;
     }
 
-    #warningBadAnchor {
-      color: red;
-      text-align: center;
+    .full-width {
       width: 100%;
     }
 
-    .full-width {
-      width: 100%;
+    .warningBox {
```

Comment:
There is also a `.warningBox` in `main.css`. Could you reuse that one?

Reply:
Will try; I've used a different one because the original one had too much `margin`

Reply:
Split it out into a new class!